tests/foreman/longrun/test_provisioning_computeresource.py

```diff
@@ -89,89 +88,6 @@ def tear_down(provisioning):
         Host.delete({'id': host['id']})
 
 
-@pytest.mark.on_premises_provisioning
-@pytest.mark.vlan_networking
-@pytest.mark.tier3
-def test_positive_provision_rhev_with_host_group(rhev, provisioning, target_sat, tear_down):
```

Comment:
Why does this test is removed, is it expected ?

Reply:
Yes, it is already covered https://github.com/SatelliteQE/robottelo/blob/6.12.z/tests/foreman/cli/test_computeresource_rhev.py#L383